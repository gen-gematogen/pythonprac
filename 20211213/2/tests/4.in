[82, 93, 46, 21, 85, 348, 47, 438, 854, 70, 93, 461, 304, 49, 0, 395, 16, 64, 42, 432, 35, 793, 6, 24, 463, 60, 10, 328, 51, 2, 377, 5, 44, 544, 69, 52, 185, 37, 19, 71, 55, 69, 340, 95, 49, 16, 95, 919]
