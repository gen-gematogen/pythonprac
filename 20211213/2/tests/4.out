[0, 2, 5, 6, 10, 16, 16, 19, 21, 24, 35, 37, 42, 44, 46, 47, 49, 49, 51, 52, 55, 60, 64, 69, 69, 70, 71, 82, 85, 93, 93, 95, 95, 185, 304, 328, 340, 348, 377, 395, 432, 438, 461, 463, 544, 793, 854, 919]
